string action #this is the task the robot needs to accomplish: "exploration" "target" "wait"
geometry_msgs/Pose target_pose